tools/release_builder.py

```diff
@@ -480,6 +480,18 @@ def _update_package_json(self, pkgdir):
         package_json['name'] = self._pkg_name
         # Update package's version to reflect the user's input
         package_json['version'] = self._pkg_version
+        # Update package's upgradesFrom/downgradesTo to reflect any package name changes
+        # due to enabling or disabling a beta bit.
+        if self._stub_universe_pkg_name != self._pkg_name:
```

Comment:
One comment on this. This effectively means that we are setting an upgrade path that has not been tested.